app/assets/javascripts/listings/components/listing/lotteryPreference.js.coffee

```diff
@@ -2,9 +2,10 @@ angular.module('dahlia.components')
 .component 'lotteryPreference',
   bindings:
     preference: '<'
+    listing: '<'
   templateUrl: 'listings/components/listing/lottery-preference.html'
 
-  controller: ['ListingDataService', (ListingDataService) ->
+  controller: ['ListingDataService', 'ListingIdentityService', (ListingDataService, ListingIdentityService) ->
```

Comment:
@software-project we can get rid of ListingIdentityService in here now too right?
```suggestion
  controller: ['ListingDataService', (ListingDataService) ->
```